homeassistant/components/konnected/sensor.py

```diff
@@ -21,16 +23,16 @@
 
 SENSOR_TYPES: dict[str, SensorEntityDescription] = {
     "temperature": SensorEntityDescription(
-        key=DEVICE_CLASS_TEMPERATURE,
+        key=SensorDeviceClass.TEMPERATURE,
```

Comment:
This shouldn't be used as a `key` (the previous neither).

Reply:
Updated to use a standard string